src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -36,8 +36,25 @@ class ConfirmationPage extends React.Component {
       <div>
         <h3 className="confirmation-page-title">Claim submitted</h3>
         <p>We usually process claims within <strong>103 days</strong>.</p>
-        <p>We may contact you if we have questions or need more information. You can print this page for your records.</p>
-
+        <p>We may contact you if we have questions or need more information.
+        You can print this page for your records.</p>
+        <p>
+          <strong>If you need us to get your private medical records from your
```

Comment:
@goldenmeanie It looks like we have access to `formData` in here, so we can show this section only if they chose the option that requires this. Should we show this information only if it's applicable?

Reply:
Oh, that's an interesting thought. Would probably make things less confusing.

Reply:
Are you talking about line 42 in particular? Or the whole thing?

Reply:
I think the message that we will contact you if necessary is generally true. But the private medical records chunk could be conditional.